example/deep-embedded-clustering/README.md

```diff
@@ -1,9 +1,18 @@
 # DEC Implementation
 This is based on the paper `Unsupervised deep embedding for clustering analysis` by  Junyuan Xie, Ross Girshick, and Ali Farhadi
 
+Abstract:
+
+Clustering is central to many data-driven application domains and has been studied extensively in terms of distance functions and grouping algorithms. Relatively little work has focused on learning representations for clustering. In this paper, we propose Deep Embedded Clustering (DEC), a method that simultaneously learns feature representations and cluster assignments using deep neural networks. DEC learns a mapping from the data space to a lower-dimensional feature space in which it iteratively optimizes a clustering objective. Our experimental evaluations on image and text corpora show significant improvement over state-of-the-art methods.
+
+
 ## Prerequisite
   - Install Scikit-learn: `python -m pip install --user sklearn`
   - Install SciPy: `python -m pip install --user scipy`
 
+## Data
+
+The script is using MNIST dataset.
+
 ## Usage
-run `python dec.py`
\ No newline at end of file
+run `python dec.py`
```

Comment:
In order to verify this file, upon running the python script `dec.py` it throws up an error for fetching the mnist data. `ConnectionResetError: [Errno 54] Connection reset by peer`
Any thoughts on that?

Reply:
This happens because sklearn is throttling some IPs or is experiencing some issues. Anyway fetch_mldata is being deprecated, I will update with using our own mxnet s3 download.